config-model/src/main/java/com/yahoo/config/model/provision/InMemoryProvisioner.java

```diff
@@ -238,6 +239,14 @@ private HostSpec retire(HostSpec host) {
         return allocation;
     }
 
+    private void dumpFreeNodes(String description) {
+        System.out.println("  Free nodes " + description + ":");
+        for (var entry : freeNodes.entrySet()) {
+            for (var node : entry.getValue())
+                System.out.println("    " + node.hostname());
+        }
+    }
+
```

Comment:
Leftover?

Reply:
Thought I'm going to need it later, but ok :-)